lib/mongoid/history.rb

```diff
@@ -19,11 +19,20 @@ class << self
       attr_accessor :modifier_class_name
       attr_accessor :current_user_method
 
-      def disable(&_block)
-        store[GLOBAL_TRACK_HISTORY_FLAG] = false
-        yield
+      def disable(&block)
```

Comment:
I feel that `disable!` and `enable!` that doesn't take a block along with `enable` and `disable` that does would be more idiomatic. In Ruby we want the developers to write "english", not pass in true/false. Then you don't need the `track(boolean)` method at all.